2-copy-of-code/lesson-16/tests/data/cartTest.js

```diff
@@ -16,6 +16,11 @@ describe('test suite: addToCart', () => {
     addToCart('e43638ce-6aa0-4b85-b27f-e1d07eb678c6');
     expect(cart.length).toEqual(1);
     expect(localStorage.setItem).toHaveBeenCalledTimes(1);
+    expect(localStorage.setItem).toHaveBeenCalledWith('cart', JSON.stringify([{
+      productId: 'e43638ce-6aa0-4b85-b27f-e1d07eb678c6',
+      quantity: 2,
+      deliveryOptionId: '1'
+    }]));

```

Comment:
no because in this case we're taking the data from localStorage that we mockec

Reply:
Thanks a lot!

Reply:
That's ok, because cart from localStorage that we mocked (in the line loadFromStorage() cart = JSON.parse(mocked getItem return), in addToCart('...') saveToStorage() was called, localStorage.setItem('cart', JSON.stringify(cart)); and setItem was mocked before).